frontend/src/store/modules/page_views/actions.js

```diff
@@ -16,4 +32,20 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [FETCH_PAGE_VIEWS_TABLE_DATA]: (context) => {
+        context.commit(SET_IS_FETCHING);
+
+        const period = getTimeByPeriod(context.state.selectedPeriod);
+
+        return fetchTableValues(period.startDate.unix(), period.endDate.unix())
+            .then(response => {
+                context.commit(SET_PAGE_VIEWS_TABLE_DATA, response);
+                context.commit(RESET_IS_FETCHING);
```

Comment:
no need to duplicate this commit twice - use just `finally` instead of catch, and perform this commit there